lib/rules/cli/4.1.yaml

```diff
@@ -1337,3 +1337,5 @@
     :openshift_version: !ruby/regexp '/^openshift v(.+)$/'
     :kubernetes_version: !ruby/regexp '/kubernetes v(.+)$/'
     :etcd_version: !ruby/regexp '/etcd (.+)$/'
+:oadm_top_node:
+  :cmd: oc adm top node
```

Comment:
should we also add to 4.2 and 4.3 rules file?

Reply:
the already exists in 4.2, 4.3 and 4.5